src/cf-layout.less

```diff
@@ -1605,6 +1605,121 @@
     }
 }
 
+/* topdoc
+  name: Featured content module
+  family: cf-layout
+  patterns:
+    - name: .featured-content-module
+      markup:
+        <section class="block
+                        block__border
+                        block__flush
+                        featured-content-module">
```

Comment:
I think this is a nice to have that we can put on the backlog to avoid this PR from having to linger any more.